plugins/appstatus.py

```diff
@@ -0,0 +1,85 @@
+import subprocess
+import sys
+import json
+from crmetrics import CRBase
+
+
+class AppStatusFinder(CRBase):
+
+    def get_app_instance_status(self, kind, instance, kubeconfig):
+        cmd = 'kubectl get %s %s -o json %s' % (kind, instance, kubeconfig)
+        out, err = self._run_command(cmd)
+        if err != "":
+            print("Something went wrong while getting app instance status.")
+            print(err)
+            exit(1)
+
+        response = json.loads(out)
+        if 'status' in response:
+            if 'helmrelease' in response['status']:
+                helm_release = response['status']['helmrelease'].strip('\n')
+                ns, name = helm_release.split(':')
+                return name, ns, True, None
+            else:
+                # an error has occurred
+                status = response['status']
+                return status, None, False, None
```

Comment:
I would suggest defining variables for the second and thrid return parameters and setting their values after lines 21 and 25. That way, by looking at the return statement it will be clear what parameters are being returned.